src/index.json

```diff
@@ -2051,16 +2051,16 @@
                             ]
                         }
                     ],
-                    "summary": "The Azure IoT extension for Azure CLI.",
-                    "version": "0.8.9"
+                    "summary": "[Deprecated: Use the 'azure-iot' extension instead.]",
```

Comment:
nice. It's better explicitly call out a date :)

Reply:
Assuming you are referring to the removal date rather than the deprecation date since "azure-cli-iot-ext" has been deprecated early in the year.

I updated the deprecated packages description with more details and a removal date:

`Deprecated: please remove 'azure-cli-iot-ext' and install the 'azure-iot' extension instead. The legacy extension 'azure-cli-iot-ext' is scheduled for removal after 9/15/2020.`